lang/en.json

```diff
@@ -1241,13 +1241,17 @@
             "buttons": {
                 "repair": "Repair",
                 "simulate": "Simulate repair",
-                "request": "Request repapir"
+                "request": "Request repapir",
+                "gmRepair": "GM repair"
             },
             "dialog": {
```

Comment:
I think the whole bloch should be moved to "repair" or something

Reply:
You mean adding a seprate key for the lowercase "repair" to reuse it in buttons?

Reply:
You put it under "dialog" but as far as I can see all the things are used for repair

Reply:
These are already under repair block - path for these strings is like `WITCHER.Repair.buttons.gmRepair`